src/vmm/src/devices/virtio/iov_deque.rs

```diff
@@ -79,8 +79,8 @@ pub enum IovDequeError {
 // ```
 //
 // This value must be a multiple of 256 because this is the maximum number of `iovec` can fit into
-// 1 memory page: 256 * sizeof(iovec) == 4096 == PAGE_SIZE. IovDeque only operates with `PAGE_SIZE`
```

Comment:
we don't claim PAGE_SIZE == 4096 anymore, do we?

Reply:
The type now can handle any host page size, so we don't need to mention 4096 pages anymore.